include/Standalone/Dialect/Tpp/TppOps.td

```diff
@@ -25,6 +25,7 @@ class StaticMemRefRankOf<list<Type> allowedTypes, list<int> ranks> :
 def TppMemRef : StaticMemRefRankOf<[AnyFloat], [1, 2]>;
 def TppPackedMemrefInput : StaticMemRefRankOf<[AnyFloat], [1, 2, 3]>;
 def TppBRGEMMemrefInput : StaticMemRefRankOf<[AnyFloat], [3]>;
+def TppBRGEMMPackedMemrefInput : StaticMemRefRankOf<[AnyFloat], [3,4]>;
```

Comment:
Can you please add verifier checks for this change? Similar with what you did for gemm.